infra/cifuzz/cifuzz.py

```diff
@@ -124,35 +171,48 @@ def build_fuzzers(  # pylint: disable=too-many-arguments,too-many-locals
   out_dir = os.path.join(workspace, 'out')
   os.makedirs(out_dir, exist_ok=True)
 
-  # Build Fuzzers using docker run.
-  inferred_url, project_builder_repo_path = (
-      build_specified_commit.detect_main_repo(project_name,
-                                              repo_name=project_repo_name))
-  if not inferred_url or not project_builder_repo_path:
-    logging.error('Could not detect repo from project %s.', project_name)
-    return False
-  project_repo_name = os.path.basename(project_builder_repo_path)
-  src_in_project_builder = os.path.dirname(project_builder_repo_path)
-
-  manual_src_path = os.getenv('MANUAL_SRC_PATH')
-  if manual_src_path:
-    if not os.path.exists(manual_src_path):
+  project_src_path = get_project_src_path()
+  if project_src_path:
+    if not os.path.exists(project_src_path):
       logging.error(
-          'MANUAL_SRC_PATH: %s does not exist. '
-          'Are you mounting it correctly?', manual_src_path)
+          'PROJECT_SRC_PATH: %s does not exist. '
+          'Are you mounting it correctly?', project_src_path)
       return False
-    # This is the path taken outside of GitHub actions.
-    git_workspace = os.path.dirname(manual_src_path)
+    git_workspace = os.path.dirname(project_src_path)
   else:
     git_workspace = os.path.join(workspace, 'storage')
     os.makedirs(git_workspace, exist_ok=True)
 
-  # Checkout projects repo in the shared volume.
+  build_integration_path = os.getenv('BUILD_INTEGRATION_PATH')
+  if build_integration_path:
+    build_integration_path = os.path.join(project_src_path,
+                                          build_integration_path)
+    logging.info('build_integration_path %s, project_src_path %s.',
+                 build_integration_path, project_src_path)
+    logging.info('Building external project.')
+    inferred_url = None
+    project_builder_repo_path = os.path.join('/src',
```

Comment:
Does /src need to be hardcoded or pulled from $SRC or some other helper function / global var.